### PR TITLE
Environment lock & docs update

### PR DESCRIPTION
**Pull request by FrickTobias**
Solves #43.

Added script to create lock files for travis to have specific versions. Since `jupyter` has post-link dependency this has to be done on a per-OS basis. Thanks to @marcelm for figuring it out and creating a workaround for it. I also took the freedom to add a `-h` message.

I've also updated the documentation because `travis` didn't identify a new build when I edited the environment files.

### CODE REVIEW

**Comment by FrickTobias** [comment]
The md5sum changed as a result of higher versions used in dependencies now when I created the locked files.

File: README.md [hunk 1]
```diff
@@ -8,27 +8,36 @@ This pipeline analyses data sequencing data from DBS-Pro experiments for protein
...
+1. Clone the git repository.
 
       git clone https://github.com/FrickTobias/DBS-Pro
 
-- Create an environment and with required dependencies. 
+2. Move into the git folder and install all dependencies in a conda environment.
 
-      conda env create -n dbspro -f environment.yml
+      cd DBS-Pro
+      conda env create -n dbspro -f DBS-Pro/environment.yml
```
Comment (pontushojer) [inline README.md]:
```suggestion
      conda env create -n dbspro -f environment.yml
```
We have moved into the `DBS-Pro` folder.

**Comment by marcelm** [comment]
Since I was mentioned and get the notifications here, I might as well comment: The `snakemake` conda package pulls in *a lot* of dependencies that are usually not needed. You can use `snakemake-minimal` instead to reduce this significantly. That’s what we do in BLR as well. If you need a specific feature that is not covered by `snakemake-minimal`, you can just list it explicitly as an additional dependency.

**Comment by FrickTobias** [comment]
I actually hadn't looked at it, but that's right, there was a lot of softwares.

I changed to `snakemake-minimal` and got 150/160 (`osx`/`linux`) instead so now it should be more in line with what you had.

**Pull request merged**